packages/snaps-controllers/src/services/AbstractExecutionService.ts

```diff
@@ -332,47 +302,30 @@ export abstract class AbstractExecutionService<WorkerType>
    *
    * Depending on the execution environment, this may run forever if the Snap fails to start up properly, therefore any call to this function should be wrapped in a timeout.
    */
-  protected abstract initEnvStream(jobId: string): Promise<{
+  protected abstract initEnvStream(snapId: string): Promise<{
     worker: WorkerType;
     stream: BasePostMessageStream;
   }>;
 
   /**
    * Terminates the Snap with the specified ID. May throw an error if
-   * termination unexpectedly fails, but will not fail if no job for the snap
+   * termination unexpectedly fails, but will not fail if no job for the Snap
    * with the specified ID is found.
    *
-   * @param snapId - The ID of the snap to terminate.
+   * @param snapId - The ID of the Snap to terminate.
    */
   async terminateSnap(snapId: string) {
-    const jobId = this.#snapToJobMap.get(snapId);
-    if (jobId) {
-      await this.terminate(jobId);
-    }
+    await this.terminate(snapId);
```

Comment:
Since this is now just a wrapper, we can probably just remove one of them?

Reply:
Good point, removed!